### ISSUE TITLE
WARNING: standard cannot locate 'standard'

### ISSUE THREAD

**Issue by chaihongjun** [SublimeLinter/SublimeLinter#1088]
When I start sublime text 3, I get the following message:

> SublimeLinter: WARNING: standard cannot locate 'standard' 

So, how can i fix it?

**Comment by kaste** [comment]
This basically just means you have not installed `standard`.

**Comment by sebastienbarre** [comment]
Was linting working with `standard` before, or are you just getting to setup `standard` with SublimeLinter?

**Comment by jesuslerma** [comment]
You need to install `standard`:
```bash
npm install standard@latest --save-dev
```

**Comment by braver** [comment]
☝️ should fix it!

**Comment by brunops** [comment]
Having this problem too, but feel like installing `standard` shouldn't be a valid solution, as now we'd be required to keep installing `standard` on every folder that has JavaScript.

Can we reopen this issue? It's pretty annoying to create a brand new file, set the syntax to JavaScript and start having this warning popup...

**Comment by jesuslerma** [comment]
You can install standard as global for your system:
```bash
npm install standard@latest
````

**Comment by braver** [comment]
Also: https://github.com/SublimeLinter/SublimeLinter/pull/1190

**Comment by kaste** [comment]
The bug will be fixed in the next update. Heads up. 😙
